docs/architecture/edge/edgehub.md

```diff
@@ -73,5 +73,5 @@ The major steps involved in this process are as follows :-
 
 EdgeHub can be configured to communicate in two ways as mentioned below:
 
-- **Through websocket protocol**: Click [here](https://github.com/kubeedge/kubeedge/tree/master/docs/proposals/quic-design.md#edgehub-connect-to-cloudhub-through-websocket-protocol) for details.
-- **Through QUIC protocol**: Click [here](https://github.com/kubeedge/kubeedge/tree/master/docs/proposals/quic-design.md#edgehub-connect-to-cloudhub-through-quic) for details.
+- **Through websocket protocol**: Click [here](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/sig-architecture/quic-design.md#edgehub-connect-to-cloudhub-through-websocket-protocol) for details.
+- **Through QUIC protocol**: Click [here](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/sig-architecture/quic-design.md#edgehub-connect-to-cloudhub-through-quic) for details.
```

Comment:
I'm unsure about the nature of these docs, but if folks are sure the contents won't change while the directory in which these contents are might, im happy to replace these with [github permanent links](https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files). i.e. the following:
- https://github.com/kubeedge/kubeedge/blob/23deff9549506bba137e9a150b040a3f37bc3e10/docs/proposals/sig-architecture/quic-design.md#edgehub-connect-to-cloudhub-through-websocket-protocol
- https://github.com/kubeedge/kubeedge/blob/23deff9549506bba137e9a150b040a3f37bc3e10/docs/proposals/sig-architecture/quic-design.md#edgehub-connect-to-cloudhub-through-quic

Then, these links won't return a 404 even if these files are moved in the future